src/sentry/migrations/0544_backfill_fix_bad_xactors.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.20 on 2023-08-01 20:51
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_xactor(apps, schema_editor):
+    ExternalActor = apps.get_model("sentry", "ExternalActor")
+
+    for xa in RangeQuerySetWrapperWithProgressBar(
+        ExternalActor.objects.filter(team_id__isnull=True, user_id__isnull=True)
```

Comment:
```suggestion
        ExternalActor.objects.filter(team_id__isnull=True, user_id__isnull=True).select_related("actor")
```

To avoid an N+1 which would add to the runtime of this migration.

Reply:
Fair, however in truth, given the selection criteria, this migration will in reality only hit about ~40 rows.

Reply:
That's fair. N will be small.